eth/beacon/db/chain.py

```diff
@@ -289,61 +290,79 @@ def persist_block_chain(
         with self.db.atomic_batch() as db:
             return self._persist_block_chain(db, blocks, self.block_class)
 
+    @classmethod
+    def _set_block_scores_to_db(
+            cls,
+            db: BaseDB,
+            block: BaseBeaconBlock
+    ) -> None:
+        # TODO: It's a stub before we implement fork choice rule
```

Comment:
I'm confused about this comment.  Leftover?

Reply:
[This](https://github.com/ethereum/py-evm/commit/ed324233c66d5722e27f9fd92266661af2562934#diff-f153130c7072bcb8bd89fe3da54eddb9R297) is the source of the comment.

Maybe this comment should be moved [to this line](https://github.com/ethereum/py-evm/pull/1701/files/53de28a49c6926e7827680113d012dc0a1fa2d23..4e5ffba6c331e31b39cec3e49e8d95efcaf90d89#diff-f153130c7072bcb8bd89fe3da54eddb9R356). What do you think?

Reply:
This was saying that we haven't implemented the fork choice rule as [the spec](https://github.com/ethereum/eth2.0-specs/blob/master/specs/core/0_beacon-chain.md#beacon-chain-fork-choice-rule) (Latest Message Driven (LMD) Greediest Heaviest Observed SubTree (GHOST)). We currently just set the `score` to the block slot number as next line (`score = block.slot`) of this comment, and by that time we were not sure if we are going to use IMD (Immediate Message Driven (IMD) or LMD.

Could add more context:
```suggestion
        # TODO: To implement the new beacon fork choice rule as the spec.
        # Current workaround is setting score to the block slot number.
```